contracts/core/modules/collect/FeeCollectModule.sol

```diff
@@ -61,21 +64,25 @@ contract FeeCollectModule is ICollectModule, FeeModuleBase, FollowValidationModu
         uint256 pubId,
         bytes calldata data
     ) external override onlyHub returns (bytes memory) {
-        (uint256 amount, address currency, address recipient, uint16 referralFee) = abi.decode(
-            data,
-            (uint256, address, address, uint16)
-        );
+        (
+            uint256 amount,
+            address currency,
+            address recipient,
+            uint16 referralFee,
+            bool followerOnly
+        ) = abi.decode(data, (uint256, address, address, uint16, bool));
         if (
             !_currencyWhitelisted(currency) ||
             recipient == address(0) ||
             referralFee > BPS_MAX ||
             amount == 0
         ) revert Errors.InitParamsInvalid();
 
-        _dataByPublicationByProfile[profileId][pubId].referralFee = referralFee;
+        _dataByPublicationByProfile[profileId][pubId].amount = amount;
         _dataByPublicationByProfile[profileId][pubId].recipient = recipient;
         _dataByPublicationByProfile[profileId][pubId].currency = currency;
```

Comment:
Generally I wouldn't mark this, but I saw you have been re-arranging those to match the order of the decoded data so maybe you would like to:

```suggestion
        _dataByPublicationByProfile[profileId][pubId].currency = currency;
        _dataByPublicationByProfile[profileId][pubId].recipient = recipient;
```

Reply:
Addressed in [03a17e5](https://github.com/aave/lens-protocol/pull/89/commits/03a17e55e3ee9ad1068ac3435a9c5b6074db2155) like the other.